layouts/community/ortho_4x12/buswerks/readme.md

```diff
@@ -1,29 +1,12 @@
-Buswerks' 4x12 Ortholinear Layout
-============================
-For use on Planck, Let's Split, and similar keyboards
+![Imgur](https://i.imgur.com/yx9tFUq.png)
 
-### Layout Overview
+# Logan's Ortho 4x12 Layout
 
-- Main Layer
-	- Enter is located on the Raise key (layer-tap)
-	- Space Cadet shift keys
-		- Lower + shift keys = Brackets[]
-		- Raise + shift keys = Braces{}
-	- The bottom left key is the Adjust modifier
-	- Escape is in the traditional Caps Lock position
-		- Holding down the escape key acts as LGUI (intended for macOS use)
-- Lower
-	- There is a numpad on the right hand (4, 5, and 6 are located under JKL/NEI/HTN)
-	- Plus and minus keys are located to the right of the numpad
-	- There are macOS desktop navigation keys and volume controls under the left hand
+This layout is what I use on my Planck. Some features of the layout:
 
-- Raise
-	- Vim-style arrow keys
-		- Below the arrow keys are home, end, pgup, and pgdn
-	- Symbols are located on the right side of the keyboard, in a somewhat logical order
-		- Symbols from the number row are in order, starting on the left pinky.
-		- Equals and underscore are in easily accessible spots, since they're commonly used in programming
-		- The ampersand is less common, so it's to the right of the equals symbol
-		
-- Adjust
-	- Function keys are located along the top row (reset is moved to the Colemak 'R' position)
\ No newline at end of file
+- Rotary encoder support (no function for clicking yet)
```

Comment:
You mean, you want the encoder to make a click sound when turning? 

If so, just add `clicky_play()` to the function, if you have Audio Clicky enabled:
https://docs.qmk.fm/#/feature_audio?id=audio-click

Reply:
No, my rotary encoder supports clicking (like when you push down on it), but I just haven't thought of what to make it do yet. If the rotary encoder is in the bottom left or you have a switch there instead, pressing it won't do anything.

Reply:
Ah okay